mvvmfx/src/main/java/de/saxsys/mvvmfx/internal/viewloader/FxmlViewLoader.java

```diff
@@ -45,9 +42,38 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(FxmlViewLoader.class);
 
+    private static void handleInjection(View codeBehind, ResourceBundle resourceBundle, ContextImpl context, ObservableBooleanValue viewInSceneProperty) {
```

Comment:
Why did you move this method? Is it possible that you have some code-format setting active that moves static methods above non-static ones? Can you please undo this because it makes reviewing actuall changes harder and distorts the git history.